### ISSUE TITLE
LINQ Provider does not accept PropertyNamingPolicy setting

### ISSUE THREAD

**Issue by shibayan** [Azure/azure-cosmos-dotnet-v3#905]
**Describe the bug**

`CosmosPropertyNamingPolicy.CamelCase` is set to `PropertyNamingPolicy`, the class property name is not converted to camel case in the query generated by LINQ Provider.

**To Reproduce**

1. Code snippet
```csharp
public class Entry
{
    public string Id { get; set; }
    public string Title { get; set; }
    public string Body { get; set; }
    public IList<string> Tags { get; set; }
    public DateTimeOffset CreatedAt { get; set; }
}

class Program
{
    static async Task Main(string[] args)
    {
        var connectionString = "";

        var cosmosClient = new CosmosClient(connectionString, new CosmosClientOptions
        {
            ConnectionMode = ConnectionMode.Direct,
            SerializerOptions = new CosmosSerializationOptions
            {
                PropertyNamingPolicy = CosmosPropertyNamingPolicy.CamelCase
            }
        });

        var container = cosmosClient.GetContainer("Blog", "Entry");

        var query = container.GetItemLinqQueryable<Entry>()
                             .Where(x => x.Tags.Contains("日記"))
                             .ToQueryDefinition();

        Console.WriteLine(query.QueryText);
    }
}
```

**Expected behavior**

Property name is converted to camel case.

```sql
SELECT VALUE root FROM root WHERE ARRAY_CONTAINS(root["tags"], "日記")
```

**Actual behavior**

Property name is not actually converted to camel case.

```sql
SELECT VALUE root FROM root WHERE ARRAY_CONTAINS(root["Tags"], "日記")
```

**Environment summary**

SDK Version: 3.3.1
OS Version: Windows 10 / .NET Core 3.0

**Comment by j82w** [comment]
@shibayan I figured out the root cause. I'm working on a fix and will do a 3.3.2 hot fix for this. It turns out the test validating this wasn't properly tagged so it wasn't running.

**Comment by j82w** [comment]
@shibayan  [3.3.2](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/) was released with the fix.
